articles/azure-functions/functions-proxies.md

```diff
@@ -47,13 +47,13 @@ With Azure Functions Proxies, you can modify requests to and responses from the
 
 By default, the back-end request is initialized as a copy of the original request. In addition to setting the back-end URL, you can make changes to the HTTP method, headers, and query string parameters. The modified values can reference [application settings] and [parameters from the original client request].
 
-Currently, there is no portal experience for modifying back-end requests. To learn how to apply this capability from proxies.json, see [Define a requestOverrides object].
+Currently, there is no portal experience for modifying back-end requests. To learn how to apply this capability from *proxies.json*, see [Define a requestOverrides object].
```

Comment:
Seems like the standard for file names in Azure Docs is to make them italics. Example: https://docs.microsoft.com/en-us/azure/azure-functions/functions-host-json